stdlib/2and3/_typeshed.pyi

```diff
@@ -0,0 +1,26 @@
+# Utility types for typeshed
+
+# This module contains various common types to be used by typeshed. The
+# module and its types do not exist at runtime. You can use this module
+# outside of typeshed, but no API stability guarantees are made. To use
+# it in implementation (.py) files, the following construct must be used:
+#
+#     from typing import TYPE_CHECKING
+#     if TYPE_CHECKING:
+#         from _typeshed import ...
+#
+# On Python versions < 3.10 and if "from __future__ import type_checking" 
+# is not used, types from this module must be quoted.
+
+import sys
+from typing import AnyStr, Text, Union
+
+# StrPath and AnyPath can be used in places where a
+# path can be used instead of a string, starting with Python 3.6.
+if sys.version_info >= (3, 6):
+    from os import PathLike
+    StrPath = Union[str, PathLike[str]]
+    AnyPath = Union[AnyStr, PathLike[AnyStr]]
```

Comment:
Shouldn't this use Unions instead of AnyStr? The else branch doesn't use a TypeVar.

Reply:
I had problems before when trying a union, but now it seems fine ...

Reply:
`PathLike[Union[str, bytes]]` does not work, it needs `PathLike[AnyStr]`. But `Union[str, bytes, PathLike[AnyStr]]` works hopefully.

Reply:
That seems to work.